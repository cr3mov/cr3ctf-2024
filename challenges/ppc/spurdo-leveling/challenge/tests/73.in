114822486378433383 941499959309288915
161 29982
91126 89393
