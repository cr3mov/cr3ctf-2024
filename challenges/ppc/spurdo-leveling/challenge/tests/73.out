3398821913028412 27215420613136 81727989829
